agent/bench-scripts/unittests

```diff
@@ -72,7 +76,7 @@ function remove_path {
 PATH=$(remove_path /opt/pbench-agent/bench-scripts ${PATH})
 PATH=$(remove_path /opt/pbench-agent/util-scripts ${PATH})
 # Allows us to intercept scp, ssh, rsync, etc.
-export PATH=${_testopt}/unittest-scripts:${_testopt}/util-scripts:${_testopt}/bench-scripts:${PATH}
+export PATH=${_testopt}/unittest-scripts:${_testopt}/util-scripts:${_testopt}/bench-scripts:${_testopt}/common/bin:${PATH}
```

Comment:
Don't we need an enhanced PYTHONPATH?

Reply:
Maybe not  here - it *is* enhanced in the trampoline code.

Reply:
I think you do need PYTHONPATH - if you don't think so, can you explain why not?